docs/core/deploying/index.md

```diff
@@ -125,8 +125,11 @@ Because your app includes the .NET Core runtime and all of your app dependencies
   > [!TIP]
   > You can reduce the size of your deployment on Linux systems by approximately 28 MB by using .NET Core [*globalization invariant mode*](https://github.com/dotnet/runtime/blob/master/docs/design/features/globalization-invariant-mode.md). This forces your app to treat all cultures like the [invariant culture](xref:System.Globalization.CultureInfo.InvariantCulture?displayProperty=nameWithType).
 
+  > [!TIP]
+  > There is a [preview Trim feature](trim-self-contained.md) that is available that can further reduce the size of your deployment but you are responsible for full end to end scenario testing for your app.
```

Comment:
Suggest we omit "but you are responsible for testing", as it's a preview feature and the developer would be responsible for such testing anyway.

```suggestion
  > There is a [preview Trim feature](trim-self-contained.md) that can further reduce the size of your deployment.
```